sdk/management/docs/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,284 @@
+## Guide for migrating to `com.azure.resourcemanager.**` from `com.microsoft.azure-mgmt-**`
+
+This document is intended for users that are familiar with an older version of the Java SDK for managment libraries (`com.microsoft.azure-mgmt-**`) ad wish to migrate their application 
+to the next version of Azure resource management libraries (`com.microsoft.azure-mgmt-**`)
+
+For users new to the Java SDK for resource management libraries, please see the [README for 'com.azure.resourcemanager.*`](http://aka.ms/azure-sdk-java-mgmt)
+
+## Table of contents
+
+* [Prerequisites](#prerequisites)
+* [Updated Maven depedencies](#updated-maven-dependencies)
+* [General Changes](#general-changes)
+  * [Authentication](#authentication)
+  * [Customized Policy](#customized-policy)
+  * [Custom HTTP Client](#custom-http-client)
+  * [Error Handling](#error-handling)
+  * [rxJava -> Reactor](#rxjava-to-reactor)
+* [Additional Samples](#additional-samples)
+
+## Prerequisites
+
+Java Development Kit (JDK) with version 8 or above.
+
+## Updated Maven dependencies
+
+The latest dependencies for resource management libraries are [available here](https://azure.github.io/azure-sdk/releases/latest/all/java.html). Please look for packages that contains "azure-resourcemanager" in the namespace.
+
+## General Changes
+
+The latest Azure Java SDK for management libraries is a result of our efforts to create a resource management client library that is user-friendly and idiomatic to the Java ecosystem.
+
+Apart from redesigns resulting from the [new Azure SDK Design Guidelines for Java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/management/docs/DESIGN.md), the latest version improves on several areas from old version.
+
+While conforming to the new guideline, we have tried our best to minimize the breaking changes. Most of the interfaces / classes / methods have stayed the same to offer user an easier migration experience.
+
+The important breaking changes are listed in the following sections:
+
+### Authentication
+
+In old version (`com.microsoft.azure-mgmt-**`), ApplicationTokenCredentials is created with all the credential parameters.
+
+In new version (`com.azure.resourcemanager.**`), in order to provide an unified authentication based on Azure Identity for all Azure Java SDKs, the authentication mechanism has been re-designed and improved to offer a simpler interface. 
+
+To the show the code snippets for the change:
+
+**In old version (`com.microsoft.azure-mgmt-**`)**
+
+```java
+ApplicationTokenCredentials = new ApplicationTokenCredentials("<ClientId>", "<TenantId>", "<ClientSecret>", AzureEnvironment.AZURE)
```

Comment:
`ApplicationTokenCredentials credential = `

Reply:
fixed

Reply:
It is still wrong, you should copy all before `=` to the code.